src/main/java/com/microsoft/jfr/FlightRecorderConnection.java

```diff
@@ -143,6 +123,12 @@ public long startRecording(RecordingOptions recordingOptions, RecordingConfigura
         }
     }
 
+    private void setOptions(RecordingConfiguration recordingConfiguration, long id) throws OpenDataException, InstanceNotFoundException, MBeanException, ReflectionException, IOException {
```

Comment:
shouldn't this be 'setConfiguration'?

Reply:
Done, I misunderstood a previous comment